standard/expressions.md

```diff
@@ -2152,27 +2168,37 @@ When an initializer target refers to an indexer, the arguments to the indexer sh
 > Point a = __a;
 > ```
 >
-> where `__a` is an otherwise invisible and inaccessible temporary variable. The following class represents a rectangle created from two points:
+> where `__a` is an otherwise invisible and inaccessible temporary variable. 
+>
+> The following class ahows a rectangle created from two points, and the creation and initialization of a rectangle instance:
 >
 > ```csharp
+> public class Point
+> {
+>     public int X { get; set; }
+>     public int Y { get; set; }
+> }
+>
 > public class Rectangle
 > {
 >     public Point P1 { get; set; }
 >     public Point P2 { get; set; }
 > }
-> ```
->
-> An instance of `Rectangle` can be created and initialized as follows:
 >
-> ```csharp
-> Rectangle r = new Rectangle
+> class A
```

Comment:
This is a trickier one in terms of avoiding declaring an extra class and method - but having *some* top-level code. Would like to discuss this in our next meeting.

Reply:
@jskeet Did we get any direction on how to handle this?

Reply:
I would suggest creating additional files for Point and Rectangle, and specifying them as required.

Reply:
Done! I removed the redundant, hard-coded 1x`Point` and 2x`Rectangle` type declarations, replacing them with the following additions to the "additional-files" folder: Point.cs, Rectangle.cs, and RectangleWithConstructorPointAllocation.cs, and using the appropriate `additionalFiles:` in the examples' metadata.